go.mod

```diff
@@ -4,38 +4,33 @@ go 1.20
 
 require (
 	github.com/Khan/genqlient v0.6.0
+	github.com/buildkite/go-pipeline v0.2.1
 	github.com/hashicorp/terraform-plugin-framework v1.3.5
 	github.com/hashicorp/terraform-plugin-framework-validators v0.12.0
 	github.com/hashicorp/terraform-plugin-go v0.18.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.28.0
 	github.com/hashicorp/terraform-plugin-testing v1.5.1
+	github.com/lestrrat-go/jwx/v2 v2.0.16
 	github.com/shurcooL/graphql v0.0.0-20181231061246-d48a9a75455f
+	gopkg.in/yaml.v3 v3.0.1
 )
 
 require (
-	github.com/Masterminds/goutils v1.1.1 // indirect
-	github.com/Masterminds/semver/v3 v3.1.1 // indirect
-	github.com/Masterminds/sprig/v3 v3.2.2 // indirect
-	github.com/agnivade/levenshtein v1.1.1 // indirect
-	github.com/alexflint/go-arg v1.4.2 // indirect
-	github.com/alexflint/go-scalar v1.0.0 // indirect
-	github.com/armon/go-radix v1.0.0 // indirect
-	github.com/bgentry/speakeasy v0.1.0 // indirect
-	github.com/google/uuid v1.3.0 // indirect
-	github.com/hashicorp/terraform-plugin-docs v0.16.0 // indirect
-	github.com/huandu/xstrings v1.3.2 // indirect
-	github.com/imdario/mergo v0.3.13 // indirect
+	github.com/Microsoft/go-winio v0.6.1 // indirect
+	github.com/buildkite/interpolate v0.0.0-20200526001904-07f35b4ae251 // indirect
+	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0 // indirect
+	github.com/goccy/go-json v0.10.2 // indirect
+	github.com/gowebpki/jcs v1.0.1 // indirect
 	github.com/kr/pretty v0.3.0 // indirect
-	github.com/mitchellh/cli v1.1.5 // indirect
-	github.com/posener/complete v1.2.3 // indirect
-	github.com/russross/blackfriday v1.6.0 // indirect
-	github.com/shopspring/decimal v1.3.1 // indirect
-	github.com/spf13/cast v1.5.0 // indirect
-	github.com/suessflorian/gqlfetch v0.6.0 // indirect
-	github.com/vektah/gqlparser v1.3.1 // indirect
-	golang.org/x/exp v0.0.0-20230809150735-7b3493d9a819 // indirect
-	golang.org/x/tools v0.8.0 // indirect
-	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
I guess all the removed line were shaken out of the dependency tree because I ran `go mod tidy` at some stage.